core/orch_test.go

```diff
@@ -1248,8 +1252,15 @@ func TestPriceInfo_ReturnsBigRat(t *testing.T) {
 	orch := NewOrchestrator(n, nil)
 
 	priceInfo, err := orch.PriceInfo(ethcommon.Address{})
-	assert.Nil(t, err)
-	assert.Zero(t, expPricePerPixel.Cmp(big.NewRat(priceInfo.PricePerUnit, priceInfo.PixelsPerUnit)))
+	assert.Nil(err)
+	assert.Zero(expPricePerPixel.Cmp(big.NewRat(priceInfo.PricePerUnit, priceInfo.PixelsPerUnit)))
+	fixedPrice, err := common.PriceToFixed(expPricePerPixel)
+	expPrice := common.FixedToPrice(fixedPrice)
+	require.Nil(err)
+	assert.LessOrEqual(priceInfo.PricePerUnit, fixedPrice)
+	assert.LessOrEqual(priceInfo.PixelsPerUnit, priceScalingFactor)
```

Comment:
Do we need an assertion about the relationship between `PixelsPerUnit` and `priceScalingFactor`? Seems that what we really care about is the test below which checks that the the `net.PriceInfo` returned and the `big.Rat` have the same values when the numerator and denominator are converted to int64. Without this assertion this test wouldn't need to know about the price scaling factor at all which is currently a detail of the `common` package since that value is not exported.

Reply:
Good point but I thought it would be nice to actually have unit tests that test the guarantee that the numerator will never be greater than fixedPrice and the denominator will never be greater than the price scaling factor.

Although I guess you could say that this is implicitly tested by the assertions below as you mention.

Reply:
fixed in ec041c2d